LinkRelay/plugin.py

```diff
@@ -302,11 +314,20 @@ def send(s):
                 self.log.info('LinkRelay:  I\'m not in in %s on %s' %
                               (relay.targetChannel, relay.targetNetwork))
             else:
-                if isPrivmsg or \
+                if isPrivmsg and 'draft/relaymsg' in targetIRC.state.capabilities_ls \
+                    and self.registryValue('useRelayMsg', relay.targetChannel) \
+                    and conf.supybot.protocols.irc.experimentalExtensions() \
+                    and targetIRC.state.channels[relay.targetChannel].isOp(targetIRC.nick):
+                    seperator = targetIRC.state.capabilities_ls['draft/relaymsg']
+                    f = functools.partial(relaymsg, relay.targetChannel,
+                    '{}{}{}'.format(args['nick'], seperator, irc.network))
```

Comment:
```
separator = targetIRC.state.capabilities_ls['draft/relaymsg']
spoofed_nick = ''.join([args['nick'], separator, irc.network)])
f = functools.partial(relaymsg, relay.targetChannel, separator
```

typo fix + uses percent-substitution + gives an explicit name to the computed value + fixes indentation